specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json

```diff
@@ -50,35 +50,47 @@
         "modelAsString": true
       }
     },
-    "ManagedIntegrationRuntime": {
-      "x-ms-discriminator-value": "Managed",
-      "description": "Managed integration runtime, including managed elastic and managed dedicated integration runtimes.",
+    "ManagedIntegrationRuntimeBase": {
+      "description": "Managed integration runtime base.",
       "type": "object",
       "allOf": [
```

Comment:
This model inherits IntegrationRuntime and should have x-ms-disciminator-value.

Reply:
This is base, why it should have discriminator? Shouldn't his be added in the children class?